app/jinja_filters.py

```diff
@@ -32,13 +38,11 @@ def strip_tags(value: str) -> Markup:
 
 @blueprint.app_template_filter()
 def format_number(value: Union[int, Decimal, float]) -> str:
-    formatted_number: str
+    locale = flask_babel.get_locale()
+
     if value or value == 0:
```

Comment:
if value is `int | Decimal | float`.... then this check should be redundant, I suspect it can actually be none and this ought to be `if value is not None`
(and types fixed, or if types are correct, delete this)

Reply:
Removed the check now based on Mebin's comment below